01-rstudio-intro.Rmd

```diff
@@ -330,8 +361,8 @@ ls()
 > instead
 >
 
-Note here that we didn't given any arguments to `ls`, but we still needed to give
-the brackets to tell R to call the function.
+Note here that we didn't given any arguments to `ls`, but we still
+needed to give the brackets to tell R to call the function.
```

Comment:
terminology: I would call these `()` symbols parentheses, these `[]` brackets, and these `{}` braces. Is that just a regional thing?